### ISSUE TITLE
doc for cognitect.aws.client.api/client shows unused :endpoint key

### ISSUE THREAD

**Issue by dchelimsky** [cognitect-labs/aws-api#161]
## Dependencies

``` clojure
{:deps {com.cognitect.aws/api {:mvn/version "0.8.484"}}}
```

## Description with failing test case

```clojure
(doc aws/client)
-------------------------
cognitect.aws.client.api/client
([{:keys [api region region-provider retriable? backoff credentials-provider endpoint endpoint-override http-client], :or {endpoint-override {}}}])
  Given a config map, create a client for specified api. Supported keys:
...
```

The `:endpoint` key shown in the `:keys` vector is never used and its presence is misleading.

**Comment by dchelimsky** [comment]
Fixed in 93c6ac08141a168fd430ce3fb043e2b6f62e8bf4

**Comment by dchelimsky** [comment]
Released in aws-api 0.8.498
